satellite/gc/gc_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gc_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storage/streams"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+)
+
+// TestGarbageCollection does the following:
+// * Set up a network with one storagenode
+// * Upload two objects
+// * Delete one object from the metainfo service on the satellite
+// * Trigger a bloom filter generation
+// * Check that pieces of the deleted object are deleted on the storagenode
+// * Check that pieces of the kept object are not deleted on the storagenode
+func TestGarbageCollection(t *testing.T) {
+	t.Skip("Garbage collection not implemented yet.")
```

Comment:
If we know the ticket that it should remove this skip I would add it to the comment, so in case that ticket gets closed without removing this skip, it will be easy to see that it was forgotten the ticket was closed without removing it.